src/visuallyCompleteCalculator.ts

```diff
@@ -136,14 +136,18 @@ class VisuallyCompleteCalculator {
       // identify timestamp of last visible change
       const end = Math.max(start, this.lastImageLoadTimestamp, this.lastMutation?.timestamp ?? 0);
 
-      const navigationEntries = performance.getEntriesByType(
+      const navigationEntry = performance.getEntriesByType(
         'navigation'
-      ) as PerformanceNavigationTiming[];
+      )[0] as PerformanceNavigationTiming;
+
       const navigationType = isBfCacheRestore
         ? 'back_forward'
+        : // @ts-ignore activationStart is an experimental property
+        navigationEntry.activationStart > 0
```

Comment:
Can we check `navigationEntry.type` instead? If it's already `prerender`, then just pass it on. I guess I am wondering if there will be able mismatch between `navigation type = prerender` and `activationStart > 0`

Reply:
I did check this, but it turns out Chrome does not actually report 'prerender' correctly.  This is in conflict with my reading of the spec.

Reply:
https://developer.chrome.com/blog/prerender-pages/#detecting-prerender-in-javascript

<img width="710" alt="image" src="https://github.com/dropbox/ttvc/assets/11243/407c5e64-4475-41f4-8eac-49f0f88e3fcc">

I guess checking `activationStart` it is then. Do we need to check `document.prerendering` for _prerendering_ page (rather than _prerendered_ page). I guess maybe we don't since we restart measurement on `prerenderingchange` anyways.

Reply:
👍 Oh, nice find.

Yes, that's right.  The `document.prerendering` value will only be true _during_ the prerender, and we never report TTVC before prerendering has finished.